lib/ddtrace/method_tracing.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'ddtrace/utils/inflector'
+
+module Datadog
+  # \MethodTracing adds simple helpers for adding
+  # tracing to your methods.
+  #
+  # = Example
+  #
+  #   extend Datadog::MethodTracing
+  #   trace_methods self, :method1, :method2
+  module MethodTracing
+    module_function
+
+    # Wrap *methods* on *klass* in a trace block
+    def trace_methods(klass, *methods)
+      methods.each { |m| trace_method(klass, m) }
+    end
+
+    def trace_class_methods(klass, *methods)
+      methods.each { |m| trace_class_method(klass, m) }
+    end
+
+    def trace_method(klass, method)
+      metric = [
+        Utils::Inflector.class_name(klass),
```

Comment:
The question is whether the class names need to be changed at all. Instead of tracing `foo.bar` and `foo.class.bar`, we could use `Foo#bar` and `Foo.bar` for instance methods and class methods, respectively.

Reply:
@marcotc - can you give your opinion on this? Is there any Datadog standard we should take into account for naming these metrics? `Foo#bar` etc is obviously more in keeping with Ruby standards.

Reply:
I talked to our internal teams and `Foo#bar` and `Foo.bar` will work 👍.

You can go ahead with this recommendation @ronan-mch. Thank you for the suggestion @bquorning!